Step02/.gitignore

```diff
@@ -0,0 +1,8 @@
+*.iml
+.gradle
+/local.properties
+/.idea/workspace.xml
+/.idea/libraries
+.DS_Store
+/build
+/captures
```

Comment:
i think you can take the .gitignore file from the assignment 1 project so those unnecessary files saved under /.idea folder would be prevented from being pushed into repo.